packages/subproviders/test/unit/redundant_rpc_subprovider_test.ts

```diff
@@ -41,6 +42,9 @@ describe('RedundantSubprovider', () => {
         const nonExistentSubprovider = new RpcSubprovider({
             rpcUrl: 'http://does-not-exist:3000',
```

Comment:
this should be deleted now, right?

Reply:
No, we still need to pass in a value to `rpcUrl` at instantiation.